pkg/virt-handler/launcher-clients/launcher-clients.go

```diff
@@ -138,9 +138,8 @@ func (l *launcherClientsManager) CloseLauncherClient(vmi *v1.VirtualMachineInsta
 	}
 
 	clientInfo, exists := l.launcherClients.Load(vmi.UID)
-	if exists && clientInfo.Client != nil {
-		clientInfo.Client.Close()
-		close(clientInfo.DomainPipeStopChan)
+	if exists {
+		clientInfo.Close()
```

Comment:
We have to call Close() and then Delete()... Wouldn't be possible to Close() on Delete() instead?

Reply:
I mean [LauncherClientInfoByVMI.Delete()](https://github.com/kubevirt/kubevirt/pull/16004/files#diff-77dd264d21980e7ab0e0f765cc8d257c369c9f32e67cde7f50581a69f5fe08eaR61) could/should call [LauncherClientInfo.Close()](https://github.com/kubevirt/kubevirt/pull/16004/files#diff-77dd264d21980e7ab0e0f765cc8d257c369c9f32e67cde7f50581a69f5fe08eaR41) leading to Delete() be enough?